images/scripts/setup_server.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
```

Comment:
@markaolson  we work for Intel?

Reply:
I didn't write this file.  I ran it through shellcheck and fixed the issues that shellcheck identified. I also renamed it from setup-server.sh to setup_server.sh so that the word separator was consistent with the other bash scripts. That's why it's showing up as a new file. I will remove the copyright notice.

Reply:
You can just add a 2nd intel copyright line as I understand.